my.css

```diff
@@ -0,0 +1,236 @@
+.header .header {
+	background: black;
+	position: relative;
+}
+
+.logo {
+	position: absolute;
+	width: 98px;
+	height: 37px;
+	left: 52px;
+	top: 36px;
+	color: white;
+	font-family: Montserrat;
+	font-style: normal;
+	font-weight: 500;
+	font-size: 30px;
+	line-height: 37px;
+}
+
+.ru {
+	display: flex;
+	justify-content: space-around;
+	position: absolute;
+	width: 31px;
+	height: 25px;
+	left: 388px;
+	top: 42px;
+	font-family: Montserrat;
+	font-style: normal;
+	font-weight: 500;
+	font-size: 20px;
+	line-height: 24px;
+}
+
+.ru__link {
+	margin-left: 25px;
+	width: 11px;
+	height: 5.5px;
+	text-decoration: none;
+	color: #ffffff;
+}
+
+.burgermenu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 60px;
+	height: 60px;
+	background-color: #dcdcdc;
+
+}
+
+.burger {
+	width: 28px;
+	height: 5px;
+	border-radius: 40%;
+	margin: 22px 0 0 15px;
+	background-color: #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)